### DISCUSSION TITLE
如何为局域网的其他设备开启代理

### DISCUSSION THREAD

**Discussion by cablsbs** [getlantern/lantern-forum-cn#246] [General]
如题，局域网内有一个nas设备，需要访问 api.themoviedb.org 这个地址，想将蓝灯的代理共享给nas服务器，如何能实现？

**Comment by MaxMKMax** [comment]
@cablsbs 您好! 当然，我们对合作持开放态度。请与我们分享更多信息。如果有什么事情确实是互利共赢的，我们很乐意讨论。谢谢你的邀请。

关于技术问题，我将与团队讨论并稍后回复您。
